adding_data_brief.qmd

```diff
@@ -23,6 +23,30 @@ schema <- get_schema()
 definitions <- austraits$definitions
 ```
 
-This chapter gives a brief overview of how to add datasets to the database. For a more detailed guide, see [adding_datasets](adding_data_long.html).
+This section gives a brief overview of how to add datasets to the database. For a more detailed guide, see [adding_datasets](adding_data_long.html).
 
-XXX add details
+It is important that all steps are followed so that our automated workflow proceeds without problems.
+
+1. Clone the `traits.build` repository from github
+2. Create a new branch in the repo, named for the new `dataset_id` in `author_year` format, e.g. `Gallagher_2014`.
+3. Create a new folder within the folder `data` with the name `dataset_id`, e.g. `Gallagher_2014`.
+4. Prepare the file `data.csv` and place it within the new folder ([details](#csv_file) here).
```

Comment:
Not sure this is the best link. That's just generic about csvs

Reply:
Have removed the links here and on the next line